### PR TITLE
Mention `--config-dir` for eosiod

### PR DESCRIPTION
**Pull request by robertkowalski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -263,6 +263,7 @@ plugin = eosio::http_plugin
 ```
 
 Now it should be possible to run `eosiod` and see it begin producing blocks.
+You can specify the location of a custom `config.ini` by passing the `--config-dir` to `eosiod`.
```
Comment (jgiszczak) [inline README.md]: Either "the `--config-dir` option and argument" or "`--config-dir`".

File: README.md [hunk 2]
```diff
@@ -896,4 +897,4 @@ make -j$( sysctl -in machdep.cpu.core_count )
 make install
 ```
 
-Your environment is set up. Now you can <a href="#runanode">build EOS and run a node</a>.
\ No newline at end of file
+Your environment is set up. Now you can <a href="#runanode">build EOS and run a node</a>.
```
Comment (jgiszczak) [inline README.md]: What's different here?  Did the line ending change and Github doesn't show it in the diff?

Comment (robertkowalski) [reply README.md]: yes, trailing white space removed

**Comment by robertkowalski** [comment]
thanks, fixed the wording

**Pull request merged**